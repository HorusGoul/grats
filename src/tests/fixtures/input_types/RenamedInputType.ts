/** @gqlType */
class SomeType {
  /** @gqlField */
  hello: string;
}

/** @gqlInput OtherName */
type MyInputType = {
  someField: string;
};
