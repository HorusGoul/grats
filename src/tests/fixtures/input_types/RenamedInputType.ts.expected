-----------------
INPUT
----------------- 
/** @GQLType */
class Query {
  /** @GQLField */
  hello: string;
}

/** @GQLInput OtherName */
type MyInputType = {
  someField: string;
};

-----------------
OUTPUT
-----------------
schema {
  query: Query
}

directive @renameField(name: String!) on FIELD_DEFINITION

type Query {
  hello: String
}

input OtherName {
  someField: String!
}